examples/reference/widgets/FileInput.ipynb

```diff
@@ -269,7 +307,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.12.2"
+   "version": "3.12.9"
   }
  },
  "nbformat": 4,
```

Comment:
When serving the below example

```python
import panel_material_ui as pmui
import panel as pn
from PIL import Image
import io

pn.extension()

def image_upload_callback(value):
    """Callback function to handle image upload."""
    if value:
        image = Image.open(io.BytesIO(value))
        return pn.pane.Image(image).servable()
    return "Upload an image"

fileinput = pmui.FileInput(
    label="Upload an image",
    accept=".png, .jpg, .jpeg",
    multiple=False,
)

pn.Column(fileinput, pn.bind(image_upload_callback, value=fileinput)).servable()
```

and uploading a .png file I see

![image](https://github.com/user-attachments/assets/c0c9504b-e62b-47c9-934f-2c928143a352)

```bash
ReferenceError: CircularProgress is not defined
```